java/src/main/java/ai/rapids/cudf/Scalar.java

```diff
@@ -133,6 +140,17 @@ public static Scalar fromUnsignedShort(Short value) {
     return Scalar.fromUnsignedShort(value.shortValue());
   }
 
+  public static Scalar fromDurationDay(int value) {
```

Comment:
Nit: suggest renaming this `durationDaysFromInt` to be consistent with the existing `timestampDaysFromInt` method.

Reply:
This still needs to be addressed.